kodilla-patterns/src/main/java/com/kodilla/patterns/strategy/BuyPredictor.java

```diff
@@ -0,0 +1,5 @@
+package com.kodilla.patterns.strategy;
+
+public interface BuyPredictor {
```

Comment:
czemu to sie znalazlo w pull requescie?

Reply:
dobre pytanie. Po wykonaniu całego modułu dodałem wszystko do lokalnego repo. Przepraszam